qt-models/weightsysteminfomodel.cpp

```diff
@@ -98,7 +98,7 @@ void WSInfoModel::updateInfo()
 	beginRemoveRows(QModelIndex(), 0, this->rows);
 	endRemoveRows();
 	rows = -1;
```

Comment:
I don't like `rows == -1` meaning no rows.

Reply:
i left this outside of the scope of this PR.